### DISCUSSION TITLE
Collaborer au projet

### DISCUSSION THREAD

**Discussion by Lou8is** [Lou8is/docs.nestjs.fr#152] [General]
Début de discussion général pour aider à la collaboration au projet :) 

## Par où commencer ?

Le mieux est de se référer à la liste d'issues et d'en choisir une qui n'est pas en cours.
Voir le projet : https://github.com/users/Lou8is/projects/1/views/1
Ou directement les issues : https://github.com/Lou8is/docs.nestjs.fr/issues

Assignez vous à une issue, cependant comme pour la plupart des projets open-source, si cette issue n'avance pas quelqu'un d'autre peut s'en charger avant vous. Vu qu'il reste beaucoup de pages à traduire, il est aujourd’hui simple de ne pas se marcher sur les pieds.

## Comment procéder

Faire un fork du projet chez vous, réaliser les modifications, puis créer une PR.

## Note : homogénéité de la traduction

De nombreux termes peuvent se traduire de façon différentes, essayons de rester cohérents dans toutes les pages.

**Comment by lisa-sce** [comment]
Discutons alors ^^, j'ai fais un fork du projet et une branche, ainsi qu'un commit en rapport avec l'issue #150  pour tester. tu me diras si ça va?

Il y a t-il qqch de déjà traduit dans le dossier CLI? si non, si ça convient je vais m'occuper de lui. je ne peux pas me mettre en Assignee sur l'issue.

**Reply by Lou8is** [reply]
ça parait nickel. 

J'ai màj mon début de discussion avec les liens vers les issues/projet. Je n'ai rien attaqué dans CLI et suis solo pour le moment, le dossier est tout à toi !

Pour assigner : pour l'instant, si tu commentes une issue je peux t'assigner dessus, je viens de le faire pour la #150 

Quand tu as finis la page, hésite pas à faire une première PR et attendre mon retour (rapide promis !), je pense que ça nous évitera des divergences de traduction.

Des règles de commit sont en place de part le repo original aussi, n'hésite pas à t'inspirer du format des miens de type `sujet(sous-sujet): message`.
